src/aktualizr_secondary/main.cc

```diff
@@ -124,12 +127,17 @@ int main(int argc, char *argv[]) {
     // storage (share class with primary)
     boost::shared_ptr<INvStorage> storage = INvStorage::newStorage(config.storage);
 
+#ifdef OPCUA_SECONDARY_ENABLED
+    AktualizrSecondaryOpcua secondary(config);
+#else
     AktualizrSecondary secondary(config, storage);
+#endif
```

Comment:
When we spoke about this on Monday, I thought that you were intending this to be a run-time configuration option?

Reply:
I used the compile time option because this is only an approach for the moment. And I was unsure about possible transport combinations, so I've risen the question. Currently, I have a little reason to set both transports simultaneously - the only one should be selected to avoid unnecessary resources utilization, I think. I see it would be suitable to introduce command line option, despite the fact it will require some handling during build image recipe processing.